src/main/java/geekbrains/java2/ht1b/Main.java

```diff
@@ -152,52 +159,52 @@ private static void updateBoard(String move, String player){
 
         //блокируем линию по горизонтали
         for(int i = 0; i < 3; i++){
-            if (board[i][0] == board[i][1] && validMove(board[i][2]) == true){
+            if (board[i][0] == board[i][1] && validMove(board[i][2])) {
                 return board[i][2];
-            } else if (board[i][0] == board[i][2] && validMove(board[i][1]) == true){
+            } else if (board[i][0] == board[i][2] && validMove(board[i][1])) {
                 return board[i][1];
-            } else if (board[i][1] == board [i][2] && validMove(board[i][0]) == true){
+            } else if (board[i][1] == board [i][2] && validMove(board[i][0])) {
                 return board[i][0];
             }
         }
 
         //блокируем линию по вертикали
         for(int j = 0; j < 3; j++) {
-            if (board[0][j] == board[1][j] && validMove(board[2][j]) == true){
+            if (board[0][j] == board[1][j] && validMove(board[2][j])) {
                 return board[2][j];
-            } else if (board[0][j] == board[2][j] && validMove(board[1][j]) == true){
+            } else if (board[0][j] == board[2][j] && validMove(board[1][j])) {
                 return board[1][j];
-            } else if (board[1][j] == board[2][j] && validMove(board[0][j]) == true){
+            } else if (board[1][j] == board[2][j] && validMove(board[0][j])) {
                 return board[0][j];
             }
         }
 
         //блокируем линию по диагонали
-        if(board[0][0] == board[1][1] && validMove(board[2][2]) == true){
+        if(board[0][0] == board[1][1] && validMove(board[2][2])) {
             return board[2][2];
-        } else if (board[0][0] == board[2][2] && validMove(board[1][1]) == true){
+        } else if (board[0][0] == board[2][2] && validMove(board[1][1])) {
             return board[1][1];
-        } else if (board[1][1] == board[2][2] && validMove(board[0][0]) == true){
+        } else if (board[1][1] == board[2][2] && validMove(board[0][0])) {
             return board[0][0];
         }
 
-        if(board[0][2] == board[1][1] && validMove(board[2][0]) == true){
+        if(board[0][2] == board[1][1] && validMove(board[2][0])) {
             return board[2][0];
-        } else if (board[0][2] == board[2][0] && validMove(board[1][1]) == true){
+        } else if (board[0][2] == board[2][0] && validMove(board[1][1])) {
             return board[1][1];
-        } else if (board[1][1] == board[2][0] && validMove(board[0][2]) == true){
+        } else if (board[1][1] == board[2][0] && validMove(board[0][2])) {
             return board[0][2];
         }
 
         //свой ход
-        for(int i = 0; i < board.length; i++){
-            for(int j = 0; j < board[i].length; j++){
-                if(board[i][j] != "1" && board[i][j] != "2"){
-                    return board[i][j];
+        for (String[] aBoard : board) {
+            for (int j = 0; j < aBoard.length; j++) {
+                if (aBoard[j] != "1" && aBoard[j] != "2") {
```

Comment:
\[Checkstyle\] ERROR: Literal Strings should be compared using equals\(\), not '\!='\.